src/test/java/core/basesyntax/RegistrationServiceTests.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistrationServiceTests {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    public static void set_up() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
```

Comment:
Consider using `@BeforeEach` instead of `@BeforeAll` for initializing your test instances. `@BeforeAll` is used for static initialization that applies to all tests once, while `@BeforeEach` is used for initialization before each test, ensuring test isolation.